src/assets/schemas/travelReimbursementSchema.json

```diff
@@ -14,10 +14,10 @@
     },
     "group_members": {
       "enum": [
-        "1",
-        "2",
-        "3",
-        "4+"
+        1,
```

Comment:
This change is incorrect. The DB will reject these values

Reply:
Hold on this comment